build.js

```diff
@@ -1,6 +1,5 @@
-import StyleDictionary from "style-dictionary";
-
-import "./helpers/custom.js";
+const StyleDictionary = require("style-dictionary");
+require("./helpers/custom");
```

Comment:
i was stupid before and didn't think to just require the entire file, was thinking i had to put it in a `const something = requires` which is why i changed the package.json